tests/contrib/litellm/test_litellm_llmobs.py

```diff
@@ -0,0 +1,277 @@
+from ddtrace._trace.pin import Pin
+from ddtrace.llmobs._llmobs import LLMObs
+import pytest
+
+from tests.contrib.litellm.utils import async_consume_stream, get_cassette_name, consume_stream, parse_response, tools
+from tests.llmobs._utils import _expected_llmobs_llm_span_event
+from tests.utils import DummyTracer
+
+
+@pytest.mark.parametrize(
+    "ddtrace_global_config",
+    [
+        dict(
+            _llmobs_enabled=True,
+            _llmobs_sample_rate=1.0,
+            _llmobs_ml_app="<ml-app-name>",
+            _dd_api_key="<not-a-real-key>",
+        )
+    ],
+)
```

Comment:
Let's remove this parameter, and instead define these values in the `litellm_llmobs` fixture?